ansible_collections/arista/avd/roles/eos_designs/templates/facts/switch/switch.j2

```diff
@@ -397,3 +397,37 @@ switch:
 {# switch.vlans #}
   vlans: {{ leaf.vlans | unique }}
 {% endif %}
+
+{# EVPN multihoming configuration on VTEPs #}
+{# switch.link_tracking #}
+{% if switch_data.combined.link_tracking.enabled is arista.avd.defined(true) %}
+{%     if switch_data.combined.link_tracking.groups is arista.avd.defined %}
+  link_tracking_groups:
+{#         Do not sort the group as the first group will be the default group for the end-points #}
+{%         for lt_group in switch_data.combined.link_tracking.groups %}
+{%             if lt_group.name is arista.avd.defined %}
+    - name: {{ lt_group.name }}
+      recovery_delay: {{ lt_group.recovery_delay | arista.avd.default(500) }}
+{%                 if lt_group.links_minimum is arista.avd.defined %}
+      links_minimum: {{ lt_group.links_minimum }}
+{%                 endif %}
+{%             endif %}
+{%         endfor %}
+{%     else %}
+  link_tracking_groups:
+    - name: LT_GROUP1
+      recovery_delay: 500
+{%     endif %}
+{% endif %}
+
+{% if switch.vtep is arista.avd.defined(true) %}
```

Comment:
This feature should not rely on vtep. It should also work on l2leaf etc.

Reply:
How about SPINE ?

Reply:
I think this is an l2 feature. Will change it accordingly.